src/components/TestRedux.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+import testReduxAction from "../Redux/Actions/testReduxActions";
+import propTypes from "prop-types";
+import { connect } from "react-redux";
+
+const { getWelcome } = testReduxAction;
+class TestRedux extends Component {
+  componentWillMount() {
+    this.props.getWelcome();
+  }
+
+  render() {
+    return (
+      <div>
+        <p>
+          message from redux: {this.props.data ? this.props.data.message : ""}
+        </p>
+      </div>
+    );
+  }
+}
+
+TestRedux.propTypes = {
+  getWelcome: propTypes.func.isRequired
+};
+
+// get data from redux and put it in props
+const mapStateToProps = state => ({
+  data: state.testRedux.data
+});
+
+// export the component while connecting to redux
+export default connect(
+  mapStateToProps,
+  { getWelcome, jj: 'jjj' }
```

Comment:
@alainburindi thanks for adding a test reducer and action. But, we won't need this when the homepage component is done, so please add comments so we remember to delete what's not needed.

Reply:
I have added the comment on testRedux Component